pkg/scheduler/apis/config/types.go

```diff
@@ -220,9 +238,15 @@ type PluginConfig struct {
 	Args runtime.Unknown
 }
 
-///////////////////////////////////////////////////////////////////////////////
-// NOTE: The following methods are intentionally left out of the staging mirror.
-///////////////////////////////////////////////////////////////////////////////
+/*
+ * NOTE: The following variables and methods are intentionally left out of the staging mirror.
+ */
+
+const (
+	// DefaultPercentageOfNodesToScore defines the percentage of nodes of all nodes
+	// that once found feasible, the scheduler stops looking for more nodes.
+	DefaultPercentageOfNodesToScore = 50
```

Comment:
we have a new constant for CustomPriorityMaxScore: https://github.com/kubernetes/kubernetes/blob/d6eba1df1466b72848255a7b6b3a9210d4ce6eca/pkg/scheduler/api/types.go#L32

Reply:
DefaultPercentageOfNodesToScore is different, do you mean to comment on another line?

Reply:
yeah, you got it :)